new PR after feedback from nature people (#3)
greenelab/nature_news_manuscript: content/04.results.md

### Creation of an Annotated News Dataset

> We have analyzed the text and citations of 16,080 news-related articles hosted on "www.nature.com".

Reviewer: What do you think of making this the total number of frontmatter articles and including the span 15 years part here.

Author: yeah, I think this makes sense changed in https://github.com/greenelab/nature_news_manuscript/pull/3/commits/27be112764daab13da1f30c1027bf6d3dd3e9327

> The articles span 15 years from 2005 to 2020, and five article types: Career Feature, News, News Feature, Technology Feature, and Toolbox.

Reviewer: Then here you could say that our primary focus is on 16,080 articles of types authored by journalists: Career Feature, News, News Feature, Technology Feature, and Toolbox.

Author: yeah, I think this makes sense changed in https://github.com/greenelab/nature_news_manuscript/pull/3/commits/27be112764daab13da1f30c1027bf6d3dd3e9327

> In our analyses, we focus primarily on journalist-written articles, but do include one analysis of scientist-written articles in the columns, Career Column and News and Views, which adds an additional 5,921 articles.

Reviewer: ```suggestion
We also include one analysis of the scientist-written news articles, Career Column and News and Views, as a comparison set of 5,921 articles.
```
I am not sure I love this wording, but something like this could help to intro the reader to the news types if they (like we) didn't get the distinctions initially.

Thirdly, looking at the proportion of unique name mentions within an articles with a specific name origin over all names mentioned in a particular year.
We used the proportion of last authors in _Nature_ with a specific name origin overall _Nature_ articles in a particular year for all three analyses.
Additionally, in our supplemental analyses, we compare against the last authorship in a random selection of _Springer_ articles.
We find that the number of quotes and unique names mentioned dramatically outnumbers the number of authors and citations (Figure {@fig:suppfig3}a).
Still, we believe that the total number of observations per data type across all years is sufficient for our analysis.
Minimum and median per data type: _Nature_ articles, (565, 679); _Springer_ articles, (1298, 1684); quotes, (4577, 6194); mentions, (3634, 5002); citations in journalist-written article, (139, 267) citations in a scientist-written article, (503, 660).

In comparing the citation rate of last author name origins in news articles, we decided to additionally analyze scientist-written articles.

> This is because scientist-written articles have many citations and could help disentangle the bias effect between scientists and journalists.

Reviewer: ```suggestion
Though fewer in number, scientist-written non-research articles have many citations making the set sufficient for analysis and provides an opportunity to measure disparities in citation patterns by author type.
```

In both journalist- and scientist-written articles, we find most names are Celtic/English or European, both with a bootstrapped estimated citation rate between 24.8-43.0% (Figure Figure {@fig:suppfig3}b),c).
East Asian names are the third highest proportion of cited names, with a bootstrapped estimated citation rate between 5.7-24.8%.
All other name origins individually account for less than 9% of total cited authors.
To compare against the composition of the last authors in _Nature_, we compare the top three most frequent name origins (Figure {@fig:fig3}b,c).
We find a slight over-enrichment for Celtic/English names and a small under-enrichment for East Asian names in scientist-written articles.

> Interestingly, we find that the under-enrichment for East Asian names in journalist-written articles is only observable from 2005 to 2009.

Reviewer: ```suggestion
Interestingly, we only observed under-enrichment for East Asian names in journalist-written articles from 2005 to 2009.
```

Furthermore, we find no significant difference for European or other name origins (Figure {@fig:suppfig4}a).
However, we do observe that European name origins are more highly cited in articles written by scientists than journalists (Figure {@fig:suppfig4}b,c). 

> We also observed the Celtic/English over-enrichment and East Asian under-enrichment when considering our subset of _Springer_ articles (Figure {@fig:suppfig4}b) for both journalist- and scientist-written articles.

Reviewer: ```suggestion
We also observed the Celtic/English over-enrichment and East Asian under-representation when considering our subset of _Springer_ articles (Figure {@fig:suppfig4}b) for both journalist- and scientist-written articles.
```

> In contrast to _Nature_, in our _Springer_ articles, we see a difference in European names, with a growing over-enrichment.

Reviewer: ```suggestion
In contrast to _Nature_, in the _Springer_ set, we see a difference in European names, with a growing over-enrichment.
```

Additionally, we see a significant difference in Arabic/Turkish/Persian name frequencies between cited authors and _Springer_ authors, however the difference is lower than observed for Celtic/English and East Asian.

> We then sought to see if quoted speaker replicated the cited authors’ over- and under-enrichment patterns.

Reviewer: ```suggestion
We then sought to determine whether or not the quoted speaker replicated the cited authors’ over- and under-enrichment patterns.
```

We found a much stronger Celtic/English over-enrichment, with quotes from Celtic/English speakers at a much higher frequency than quotes from European speakers (Figure {@fig:suppfig3}d).

> Additionally, we also found a much stronger under-enrichment of quotes from East Asian speakers (Figure {@fig:suppfig3}b), with never more than 7.9% of quotes even though they constitute between 5.7-24.8% of cited authors.

Reviewer: ```suggestion
Additionally, we also found a much stronger depletion of quotes from East Asian speakers (Figure {@fig:suppfig3}b), with never more than 7.9% of quotes even though they constitute between 5.7-24.8% of cited authors.
```

> When we again compare against last authorship in _Nature_, we find the same patterns observed in the previously described citation analysis with all name origins except for East Asian and Celtic/English closely matching the background rate (Figure {@fig:fig3}d).

Reviewer: ```suggestion
When we again compare against last authorship in _Nature_, we observe patterns consistent with the citation analysis with all name origins except for East Asian and Celtic/English closely matching the background rate (Figure {@fig:fig3}d).
```

Similarly, we find the same patterns in quoted speakers with East Asian, Celtic/English, and Arabic/Turkish/Persian name origins when comparing against the _Springer_ background as we did in the previous citation analysis (Figure {@fig:suppfig4}d).
In addition, we also find an under-enrichment of Hispanic, South Asian, and Hebrew names when comparing against the _Springer_ background set.

> Since quoted speakers may be biased towards English speakers, we expanded our analysis to also include sources that were possibly paraphrased and not directly quoted. 

Reviewer: ```suggestion
It was possible that journalists preferentially paraphrased speakers with some name origins.
To examine whether or not this was driving the results, we expanded our analysis to also include sources that were possibly paraphrased and not directly quoted. 
```

> To do this, we identified all unique names that were identified in an article.

Reviewer: ```suggestion
To do this, we identified all unique names that appeared in an article, which we term mentions.
```
